tests/system/python/service/test_service_notification.py

```diff
@@ -0,0 +1,199 @@
+        data = {"name": SERVICE_NAME,
+                "type": "notification",
+                "enabled": "true"
+                }
+        conn.request("POST", '/foglamp/service', json.dumps(data))
+        r = conn.getresponse()
+        assert 200 == r.status
+        r = r.read().decode()
+        jdoc = json.loads(r)
+        assert 2 == len(jdoc)
+        assert SERVICE_NAME == jdoc['name']
+
+        remove_directories("/tmp/foglamp-service-{}".format(SERVICE))
+
+    def _install_notify_plugin(self, notify_branch):
+        try:
+            subprocess.run(["$FOGLAMP_ROOT/tests/system/python/scripts/install_c_plugin {} notify {}".format(
+                notify_branch, NOTIFY_PLUGIN)], shell=True, check=True)
+        except subprocess.CalledProcessError:
+            assert False, "{} installation failed".format(NOTIFY_PLUGIN)
+
+    def _get_result(self, foglamp_url, path):
+        conn = http.client.HTTPConnection(foglamp_url)
+        conn.request("GET", path)
+        r = conn.getresponse()
+        assert 200 == r.status
+        r = r.read().decode()
+        jdoc = json.loads(r)
+        return jdoc
+
+    def _verify_service(self, foglamp_url, status):
+        jdoc = self._get_result(foglamp_url, '/foglamp/service')
+        svc = jdoc['services'][2]
+        assert SERVICE_NAME == svc['name']
+        assert SERVICE.capitalize() == svc['type']
+        assert status == svc['status']
+
+    def _verify_audit_log_entry(self, foglamp_url, path):
+        jdoc = self._get_result(foglamp_url, path)
+        audit_detail = jdoc['audit'][0]
+        assert 1 == jdoc['totalCount']
+        assert 'INFORMATION' == audit_detail['severity']
+        assert SERVICE_NAME == audit_detail['details']['name']
+
+    def test_service(self, reset_and_start_foglamp, service_branch, foglamp_url, wait_time, retries, remove_directories):
+        self._configure_and_start_service(service_branch, foglamp_url, remove_directories)
+
+        retry_count = 0
+        # only 2 services is being up by default i.e core and storage
+        default_registry_count = 2
+        service_registry = default_registry_count
+        while service_registry != 3 and retry_count < retries:
+            svc = self._get_result(foglamp_url, '/foglamp/service')
+            service_registry = svc['services']
+            retry_count += 1
+            time.sleep(wait_time * 2)
+
+        if len(service_registry) == default_registry_count:
+            assert False, "Failed to start the {} service".format(SERVICE)
+
+        self._verify_service(foglamp_url, 'running')
+
+        self._verify_audit_log_entry(foglamp_url, '/foglamp/audit?source=NTFST')
+
+    def test_get_default_notification_plugins(self, foglamp_url, remove_directories):
+        remove_directories(os.environ['FOGLAMP_ROOT'] + '/plugins/notificationDelivery')
+        remove_directories(os.environ['FOGLAMP_ROOT'] + 'cmake_build/C/plugins/notificationDelivery')
```

Comment:
from both, just to ensure that ['FOGLAMP_ROOT'] can be for make or make  install. I think make scenario is enough or we expect system tests to be run with production install as well? /cc @Singhal-Vaibhav